src/main/scala/ai/starlake/extract/JDBCSchemas.scala

```diff
@@ -96,7 +96,8 @@ case class JDBCSchema(
   fetchSize: Option[Int] = None,
   stringPartitionFunc: Option[String] = None,
   fullExport: Option[Boolean] = None,
-  sanitizeName: Option[Boolean] = None
+  sanitizeName: Option[Boolean] = None,
+  where: Option[String] = None
```

Comment:
In load, I think we named this attribute `filter` and not `where` and they have the same feature.

Reply:
You're right. 
Change made